deep_qa/layers/attention/gated_attention.py

```diff
@@ -124,3 +123,8 @@ def call(self, inputs, mask=None):
             raise ConfigurationError("Invalid gating function "
                                      "{}, expected one of {}".format(self.gating_function,
                                                                      GATING_FUNCTIONS))
+
+    def get_config(self):
+        config = {'gating_function': self.gating_function}
+        base_config = super(GatedAttention, self).get_config()
+        return dict(list(base_config.items()) + list(config.items()))
```

Comment:
i think so, but keras uses this pattern across their codebase. https://github.com/fchollet/keras/blob/3d400116b95074f454605605f5339b90e28b9247/keras/layers/advanced_activations.py#L37